static_frame/test/unit/test_frame.py

```diff
@@ -14560,7 +14560,7 @@ def test_frame_bloc_sel_a(self) -> None:
     #---------------------------------------------------------------------------
     def test_sizeof_a(self) -> None:
         f = ff.parse('s(3,4)')
-        self.assertEqual(getsizeof(f), 976)
+        self.assertEqual(getsizeof(f), 1336)
```

Comment:
These are the kinds of tests that might be very fragile and fail on Windows or different versions of Python / NumPy...

Reply:
e.g.,: https://github.com/InvestmentSystems/static-frame/actions/runs/3038997283/jobs/4893403939

Reply:
I will look into a better way of making these work on different operating systems/versions of python.